theme/navigateregions.less

```diff
@@ -0,0 +1,72 @@
+/* Import all components */
+@import 'themes/default/globals/site.variables';
+@import 'themes/pxt/globals/site.variables';
+
+/* Reference import */
+@import (reference) "semantic.less";
+
+.navigate-regions-container {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0,0,0,0);
```

Comment:
For the various transparency backgrounds, I'd recommend using the `pxt-neutral-alpha*` theme variables:
```
"pxt-neutral-alpha0": "rgba(0, 0, 0, 0)",
"pxt-neutral-alpha10": "rgba(0, 0, 0, 0.1)",
"pxt-neutral-alpha20": "rgba(0, 0, 0, 0.2)",
"pxt-neutral-alpha50": "rgba(0, 0, 0, 0.5)",
"pxt-neutral-alpha80": "rgba(0, 0, 0, 0.8)",
```

The main benefit is for dark themes, where these will fade white instead of black (which usually isn't very visible against a dark theme).

If 0% 10%, 20%, 50%, and 80% don't cover your cases, you can add more values to the base theme, but it'll probably be easier to just use whichever one is closest to your current value.